ssv/spectest/tests/runner/pre_consensus_justifications/invalid_slot.go

```diff
@@ -32,6 +29,22 @@ func InvalidSlot() tests.SpecTest {
 		return signed
 	}
 
+	msgF := func(obj *types.ConsensusData, id []byte) *qbft.SignedMessage {
+		// change slot
+		if len(obj.PreConsensusJustifications) > 0 {
+			obj.PreConsensusJustifications[0].Message.Slot = testingutils.TestingDutySlot2
+		}
+		return signedMsg(obj, id)
+	}
+
+	msgFV := func(obj *types.ConsensusData, id []byte) *qbft.SignedMessage {
+		// change slot
+		if len(obj.PreConsensusJustifications) > 0 {
+			obj.PreConsensusJustifications[0].Message.Slot = testingutils.TestingInvalidDutySlotV(obj.Version)
+		}
+		return signedMsg(obj, id)
+	}
```

Comment:
we still need both?

Reply:
theoretically not, but if we want to keep the same design for all the versioned examples in current PR, so yes